src/types.ts

```diff
@@ -1 +1,4 @@
-export type ThemeColorOption = "white" | "gray";
+type ThemeColorOption = "white" | "gray" | "blue";
```

Comment:
Currently, this type really just covers backgrounds, and I don't believe we have any instance of a component with a blue background option. Is this something you were just trying out before creating the `TitleColorOption` type? If so, and if the `blue` option isn't being used anywhere, could you remove the `blue` option from this list?